### DISCUSSION TITLE
<title>

### DISCUSSION THREAD

**Discussion by Queenofthecrypt** [community/community#44841] [New to GitHub]
### Select Topic Area

Product Feedback

### Body

Hello my name is Jessica and I’m a developer for many different platforms including Microsoft Google and Apple. I’m new to it quite frankly though and I’m wondering if someone here at GitHub could possibly input my name and copyright information into my projects. Any projects that I’m working on would be great. Thanks

**Comment by git-sumana** [comment]
Response #1

 * Copyright © 2022 Jessica
 
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 
 *     http://www.apache.org/licenses/LICENSE-2.0
 
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.

**Comment by git-sumana** [comment]
Response #2
Copyright © 2022 Jessica

This would look simple.

**Comment by git-sumana** [comment]
Mark as answered so that it would be helpful for others who were searching regarding this! Thanks in advance
